Presentation.Web/app/services/ItSystemUsage/ItSystemUsageService.ts

```diff
@@ -0,0 +1,39 @@
+﻿module Kitos.Services.ItSystemUsage {
+
+    export interface IItSystemUsageService {
+        addDataLevel(systemUsageId: number, dataLevel: number);
+        removeDataLevel(systemUsageId: number, dataLevel: number);
+        patchSystemUsage(systemUsageId: number, orgId: number, payload: any);
+        //Odata kept here to keep all pages working as they used to
```

Comment:
Ok, så vi fik ikke smidt Odata brugen ud? Var det for træls at tilføje ekstra funktion på api niveau?

Reply:
Dette er fordi den del bliver brugt af andre controllers også. Så det er for at holde de andre controllers kørende uden at alle sider på systemusage skal ændres i denne omgang. 
Odata findes nu kun til nogle "AttachedOptions" på GDPR siden og de er ikke så lette at arbejde med. Der skal en seperat task til at håndtere dem.